### PR TITLE
Add Versionable.detach() for creating a copy of a model. See #68

### PR DESCRIPTION
**Pull request by ezheidtmann**
Name suggested by @brki in #68. 

I have not yet tested this with ForeignKey or ManyToManyField relations.

### CODE REVIEW

**Comment by brki** [comment]
This was merged in v1.4.5
